### ISSUE TITLE
Quick Share App

### ISSUE THREAD

**Issue by Skizzy24** [0x192/universal-android-debloater#736]
A536B

**Packages:**

```
com.samsung.android.app.sharelive
...
```

- [I removed all those packages on my phone] 

## Document each package the best you can

**List**: ``OEM` (manufacturer)

**Removal**: `Recommended`

> Description. its Quick Share on the Galaxy Store, doesn't break anything if you remove it. (Obv you can't use quick share)

**Comment by AnonymousWP** [comment]
I'd say it's `Advanced` to be honest, not `Recommended`, see: https://github.com/0x192/universal-android-debloater/wiki/FAQ#how-are-the-recommendations-chosen

**Comment by Skizzy24** [comment]
You can still redownload it from the store tho

**Comment by AnonymousWP** [comment]
> You can still redownload it from the store tho

Yeah, but on the other hand, it might be a useful feature, no? Because `Recommended` also says: "Pointless or outright negative packages".

**Comment by Skizzy24** [comment]
Imo its pointless because nearby share exists and it has a bigger ecosystem

**Comment by AnonymousWP** [comment]
> Imo its pointless because nearby share exists and it has a bigger ecosystem

Alright, fair, I'll change it to `Recommended`.
